jcl/source/common/JclSysInfo.pas

```diff
@@ -4293,6 +4294,28 @@ function GetWindowsReleaseName: String;
     Result := '';
 end;
 
+function GetWindowsReleaseCode: String;
+begin
+  // Looks much like the 'GetWindowsReleaseCodeName', except for the Windows 10 versions
+  // prior to Release Id 1903 - those have a different 'code' vs the 'code name'.
+  if IsWin10 then
+  begin
+    case GetWindowsReleaseId of
+      1903:
+        Result := '19H1';
+      1909:
+        Result := '19H2';
+      2004:
+        Result := '20H1';
+      2009:
+        Result := GetWindowsDisplayVersion;
+    else
+      Result := IntToStr(GetWindowsReleaseId);
+    end
+  end else
```

Comment:
[coding style] This else should be on it own line

Reply:
Placed `else` on new line